### PR TITLE
docs(onboarding): add test command decision tree

### LINKED ISSUE #1772: docs(onboarding): add test command decision tree
## Description
Implement an LLM-friendly, independently shippable backlog item for **add test command decision tree**. This issue should stay narrow enough for one agent/PR while producing durable behavior, tests, and documentation where appropriate.

## Affected locations
- docs/onboarding and contributor guides
- CLI/help output or onboarding scripts
- PM/worker handoff templates

## Impact
Without this work, future agents and operators will keep rediscovering the same gap, creating avoidable coordination cost, reliability risk, or ambiguous handoffs.

## Recommendation
Implement the scoped behavior with deterministic tests, concise docs, and structured output that PM/liveness tooling can consume.

## Goals
Deliver add test command decision tree as a discrete capability or hardening change.
Make the behavior deterministic enough for autonomous workers to verify.
Keep the implementation small enough for a one-issue worker and one PR.
Capture operator/user-facing behavior in docs or help text when applicable.

## Expected outcomes
add test command decision tree is available, documented, and test-backed.
Future PM/worker handoffs can point at this issue without needing extra interpretation.
Failure modes are explicit and produce actionable messages instead of silent drift.
The change does not broaden into adjacent backlog work or unrelated refactors.

## Suggested implementation slices
Inventory the current code/docs paths that own this behavior.
Add or update the narrow model/config/schema needed for the feature or guardrail.
Implement the smallest runtime/docs change that satisfies the acceptance criteria.
Add focused fixtures/tests for success, failure, and edge cases.
Update docs/help output and handoff notes so later agents can use it.

## Acceptance criteria
- [ ] There is a dedicated implementation or documentation change for add test command decision tree.
- [ ] At least one automated test, fixture, or deterministic verifier covers the intended behavior.
- [ ] At least one negative/edge case is covered so the failure mode is explicit.
- [ ] Docs/help text or inline operator guidance explains how to use or interpret the behavior.
- [ ] The PR description links this issue and summarizes verification evidence.
- [ ] No unrelated production behavior is changed outside the stated scope.

## LLM handoff notes
- Treat this as a one-issue/one-PR unit.
- Start by searching for existing implementations and tests before adding new surfaces.
- Prefer structured outputs and fixtures over prose-only behavior.
- If this overlaps an existing open issue, comment with evidence and stop for PM deduplication instead of creating an unscoped implementation.
- Do not perform broad refactors or change unrelated worker/PM behavior while implementing this ticket.

## Source
Created by the 2026-07-11 fourth strategic backlog expansion run requesting 100 issues. Topic: **Onboarding**.

### PR DESCRIPTION
**Pull request by djm204**
## Summary
- add a dedicated onboarding test command decision tree for docs, root guardrails, packages, contracts, opt-in suites, and CI-equivalent handoffs
- link the guide from ONBOARDING and the quickstart verification section
- add a root docs verifier covering live root script names and negative/edge cases

## Verification
- npm run test:root -- tests/docs-issue-1772.test.ts
- npm run test:root -- tests/docs-issue-1772.test.ts tests/docs-issue-1773.test.ts

Closes #1772

### CODE REVIEW

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/test-command-decision-tree.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+   - Build the shared types package, then run the consuming package checks named by the ownership manifest:
+     ```bash
+     npm run build --workspace @franken/types
+     npm run typecheck
+     ```
+
+5. **Did you change integration, eval, E2E, or live-benchmark behavior?**
+   - Use the explicit opt-in suite for that surface instead of assuming default `npm test` covers it:
+     ```bash
+     npm run test:integration
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/test-command-decision-tree.md:51]: Addressed in commit 12be5ef1: the guide now recommends package build gates, package-specific integration commands, root E2E test commands, and the separate ci:test:e2e gate for CI-equivalent handoffs.

File: docs/onboarding/test-command-decision-tree.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+     npm run test:eval
+     npm run test:e2e
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/test-command-decision-tree.md:53]: Addressed in commit 12be5ef1: the guide now recommends package build gates, package-specific integration commands, root E2E test commands, and the separate ci:test:e2e gate for CI-equivalent handoffs.

File: docs/onboarding/test-command-decision-tree.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+     npm run test:live:bench
+     ```
+
+6. **Are you preparing a CI-equivalent local handoff?**
+   - Use the aggregate CI test target after the narrower checks pass:
+     ```bash
+     npm run test:ci
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/test-command-decision-tree.md:69]: Addressed in commit 12be5ef1: the guide now recommends package build gates, package-specific integration commands, root E2E test commands, and the separate ci:test:e2e gate for CI-equivalent handoffs.

File: docs/onboarding/test-command-decision-tree.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+     ```bash
+     npm run lint
+     npm run typecheck
+     ```
+
+3. **Did you change one package's runtime or tests?**
+   - Run that package's targeted test and typecheck scripts:
+     ```bash
+     npm test --workspace @franken/<package>
+     npm run typecheck --workspace @franken/<package>
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/test-command-decision-tree.md]: Addressed in commit 12be5ef1: the guide now recommends package build gates, package-specific integration commands, root E2E test commands, and the separate ci:test:e2e gate for CI-equivalent handoffs.

**Comment by djm204** [comment]
@codex review
